ide/static/js/netLayout_vertical.js

```diff
@@ -33,6 +33,7 @@ function allocatePosition(layerId, preferredPosition){
   if (!map.hasOwnProperty(preferredPosition[1])) {
     map[preferredPosition[1]] = [];
   }
+  map[0]=[];
```

Comment:
Added because without it,an error was showing like map[0](map[preferredPosition[1]-1) not defined.